cmd/admin-handlers.go

```diff
@@ -284,6 +284,73 @@ func (a adminAPIHandlers) ServerInfoHandler(w http.ResponseWriter, r *http.Reque
 	writeSuccessResponseJSON(w, jsonBytes)
 }
 
+// ServerDrivePerfInfo holds informantion about address and write speed of all drives in a single server node
+type ServerDrivePerfInfo struct {
+	Addr       string            `json:"addr"`
+	DrivesPerf []DriveWriteSpeed `json:"drivesPerf"`
```

Comment:
Perf should contain Write and Read speed as we might do Reads as well in future @nitisht